Bump dev dependencies

compulim opened the pull request:
## Changelog Entry

### Changed
-  `*`: Bumps all dev dependencies to latest version, by [@compulim](https://github.com/compulim), in PR [#2182](https://github.com/microsoft/BotFramework-WebChat/pull/2182), notably
   -  [`@babel/*@7.5.4`](https://www.npmjs.com/package/@babel/core)
   -  [`jest@24.8.0`](https://www.npmjs.com/package/jest)
   -  [`lerna@3.15.0`](https://www.npmjs.com/package/lerna)
   -  [`typescript@3.5.3`](https://www.npmjs.com/package/typescript)
   -  [`webpack@4.35.3`](https://www.npmjs.com/package/webpack)
-  `*`: Bumps [`@babel/runtime@7.5.4`](https://www.npmjs.com/package/@babel/runtime), by [@compulim](https://github.com/compulim), in PR [#2182](https://github.com/microsoft/BotFramework-WebChat/pull/2182)
-  `*`: Bumps Docker container for headless Chrome to `selenium/standalone-chrome:3.141.59-radium`, by [@compulim](https://github.com/compulim), in PR [#2182](https://github.com/microsoft/BotFramework-WebChat/pull/2182)
-  `*`: Moves from [`babel-plugin-version-transform`](https://www.npmjs.com/package/babel-plugin-version-transform) to [`babel-plugin-transform-inline-environment-variables`](https://www.npmjs.com/package/babel-plugin-transform-inline-environment-variables), by [@compulim](https://github.com/compulim), in PR [#2182](https://github.com/microsoft/BotFramework-WebChat/pull/2182)

## Description

Bumps all dev dependencies to latest version before we start the development of next version. Production dependencies will be evaluated individually in other PRs.

## Specific Changes

- Bumps dependencies for `bundle`, `component`, `core`, `embed`, and `playground` packages
- Updates `docker-compose.yml` to use latest version of headless Chrome
- Moves from [`babel-plugin-version-transform`](https://www.npmjs.com/package/babel-plugin-version-transform) to [`babel-plugin-transform-inline-environment-variables`](https://www.npmjs.com/package/babel-plugin-transform-inline-environment-variables) because the original plugin is 2 years old and failing `npm audit` without a viable fix, thus, we need to move away from it

---

corinagum commented on packages/.prettierignore:
```diff
@@ -0,0 +1 @@
+**/**/*.md
```
do we really want to entirely ignore `.md` files? I made the command prettier-readmes so that we can improve the readmes manually.

compulim replied:
I copied it from the original `package.json`, where it was originally.

```json
  "lint-staged": {
    "linters": {
      "*.{js,jsx,md,ts,tsx}": [
        "prettier --write packages/{bundle,component,core,embed,playground}/src/**/*.{js,jsx,ts,tsx}",
        "prettier --write __tests__/**/*.{js,jsx,ts,tsx}",
        "git add"
      ]
    },
    "ignore": [
      "**/**/*.md"
    ]
  },
```

After upgrading `lint-staged`, it removed `ignore` options and suggested to move it to `.prettierignore` instead. See it here, https://github.com/okonet/lint-staged#ignoring-files.

compulim replied:
I know you have Prettier for Markdown and looks weird to me too.

But looks like before my bump, Prettier is disabled for Markdown.

corinagum replied:
this ignores prettier for the lint-staged command, which happens on commit. I'm just not sure if .prettierignore conflicts with the manual command `npm run prettier-readmes`

compulim replied:
Oh sorry, didn't know there is another NPM script.

I should remove `.prettierignore` and let Husky run on paths without Markdown. Hold on.

corinagum replied:
can you also add the ignore back to the package.json files?

compulim replied:
I can't, the option is deprecated from `lint-staged`. That's why the recommend using `.prettierignore`.

corinagum replied:
ahhhh

compulim commented:
> * I can't immediately tell what the difference is on the screencaptures, could you explain?

Since we updated Docker container for Chrome, the render behavior is slightly different. Like 1px shifted, so I updated those images.

This is the sample of one of the diff.

![image](https://user-images.githubusercontent.com/1622400/61144676-5fba5580-a48a-11e9-839b-5d3af8588e80.png)
